### PR TITLE
Zig Zag level Order Traversal in Tree

### PR DESCRIPTION
**Pull request by Pranshu321**
## Related Issue
- #1314 

## Proposed Changes
-  Cpp file added
- Readme Updated

## Additional Info
In this, I have explained in a very simple how a zig-zag traversal is performed and how it should be approached.

## Checklist

- [x] ✅ My code follows the code style of this project.
- [x] 📝 My change requires a change to the README.
- [x] 🎀 I have updated the README accordingly.
- [x] 👀 I have read the [**CONTRIBUTING**](https://github.com/Lakhankumawat/LearnCPP/blob/main/Contributing.md) document.
- [x] ✨ I have added tests to cover my changes.
- [x] 🚩 All new and existing tests passed.
- [x] 🌟 ed the repo

#1314 

In any GCC compiler.

- [x] New feature (non-breaking change which adds functionality)

## Output Screenshots
| Screenshot #1      |
| ----------- | 
| Output Screenshot| 
image

### CODE REVIEW

**Comment by Mrjoy832** [comment]
Resolve conflict

**Comment by Pranshu321** [comment]
> Resolve conflict

Resolved Sir @Mrjoy832

**Comment by Mrjoy832** [comment]
Resolve Branch Conflict

**Comment by Pranshu321** [comment]
> Resolve Branch Conflict

Now check sir , @Mrjoy832

File: T-Tree/T-TreeTraversals/Zig_Zag_level_Order_Traversal.cpp [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+    root->left->left = new node(7);
+    root->left->right = new node(6);
+    root->right->left = new node(5);
+    root->right->right = new node(4);
+    cout << "\n\n--------------------------------------\n\n";
+    cout << " ZigZag Order traversal of tree is :-\n\n";
+    cout << " ";
+    zigzagTraversal(root);
+    cout << "\n\n--------------------------------------\n\n";
+}
```
Comment (Mrjoy832) [inline T-Tree/T-TreeTraversals/Zig_Zag_level_Order_Traversal.cpp:84]: Add some comments

**Review by Mrjoy832 — CHANGES_REQUESTED** [review]

File: T-Tree/T-TreeTraversals/Zig_Zag_level_Order_Traversal.cpp (hunk 1 above)
Comment (Pranshu321) [reply T-Tree/T-TreeTraversals/Zig_Zag_level_Order_Traversal.cpp]: > Add some comments

Sir added , Sorry for delay @Mrjoy832

**Comment by Mrjoy832** [comment]
https://github.com/Lakhankumawat/LearnCPP/blob/main/T-Tree/T-TreeTraversals/zigzag_levelorderTraversal.cpp
It's already merged ..Sorry but I am closing Yours

**Pull request closed without merging**